docs/_sidebar.md

```diff
@@ -1,7 +1,10 @@
 - [Overview](home)
 - [Getting Started](quickstart)
 - [Onboarding Public Officers using non-SE Device](onboard-public-officers-using-non-se-machines)
-  - [Log in to TechPass Portal](log-into-techpass-portal-po-non-se)
+- Manage security verification
+  - [Set up security verification for WOG accounts](manage-security-verification-for-wog-account)
```

Comment:
Suggestion:
[Set up security verification for WOG account] < remove s from accounts
[Reset security verification for WOG account] < Re set up seems abit funny )

Reply:
:-), got it overlooked this!